host_core/native/hostcore_wasmcloud_native/src/wasmruntime.rs

```diff
@@ -103,10 +169,13 @@ pub fn on_load(env: Env) -> bool {
 #[rustler::nif(name = "runtime_new")]
 pub fn new<'a>(
     env: rustler::Env<'a>,
-    _config: ExRuntimeConfig,
+    ExRuntimeConfig { host_id }: ExRuntimeConfig,
```

Comment:
This is never actually used anymore, right?

Reply:
we still need the `host_id` field, it's passed to the `ElixirHandler` constructor